cli/src/cli_util.rs

```diff
@@ -2707,6 +2708,24 @@ fn resolve_aliases(
             real_commands.insert(alias.to_string());
         }
     }
+    for alias in aliases_map.keys() {
+        if real_commands.contains(alias) {
+            return Err(user_error(format!(
+                "Cannot define an alias that overrides the built-in command '{alias}'"
+            )));
+        }
+    }
+
+    let mut resolved_alias_map = HashMap::new();
```

Comment:
nit: rename to `all_aliases_map` or something. `resolved_aliases: HashSet` contains only expanded aliases, but the `_map` doesn't.

Reply:
hmm, i don't think "all" is very descriptive - i renamed `resolved_aliases` to `expanded_aliases` and the function to `expand_aliases`, does that sound ok?

Reply:
Yes, `expanded_aliases` sounds good, thanks.